testsuite/lees_edwards.py

```diff
@@ -1,34 +1,42 @@
 #!/usr/bin/env python
 
+
 from __future__ import print_function
 
 import espressomd as md
 import unittest as ut
 
+
 @ut.skipIf(not md.has_features(['LEES_EDWARDS']),
-'Features not available, skipping test!')
+  'Feature not available, skipping test!')
 
-class Lees_Edwards_test(ut.TestCase):
+class lees_edwards_test(ut.TestCase):
```

Comment:
Sorry that is still wrong... it should be names "LeesEdwardsTest" or similar.

Reply:
Sorry for the effort, should be correct now.